packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts

```diff
@@ -13,6 +18,7 @@ export const updateVscode: Rule = async () => {
       const settings = tree.exists(vscodeSettingsPath) ? (tree.readJson(vscodeSettingsPath) || {}) as JsonObject : {};
       settings['eslint.useFlatConfig'] = true;
       settings['editor.defaultFormatter'] = 'dbaeumer.vscode-eslint';
+      settings['editor.foldingImportsByDefault'] = true;
```

Comment:
Thanks to `unused-imports/no-unused-imports` we can collapse all imports by default :)